editor.planx.uk/src/ui/editor/ComponentTagSelect.tsx

```diff
@@ -16,42 +17,65 @@ interface Props {
   onChange: (values: NodeTag[]) => void;
 }
 
-const renderOption: AutocompleteProps<
-  NodeTag,
-  true,
-  true,
-  false,
-  "div"
->["renderOption"] = (props, tag, { selected }) => (
-  <ListItem {...props}>
-    <CustomCheckbox aria-hidden="true" className={selected ? "selected" : ""} />
-    {TAG_DISPLAY_VALUES[tag].displayName}
-  </ListItem>
-);
+export const ComponentTagSelect: React.FC<Props> = ({ value, onChange }) => {
```

Comment:
Zustand has a non-hook API for use outside of React components, which means you don't have to nest these within `ComponentTagSelect` - 

```ts
const isPlatformAdmin = useStore.getState().user?.isPlatformAdmin;
```